docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -84,13 +84,19 @@ Examples:
 |
 |List of relative URLs, a seed URL is a URL that is fetched by the crawler to extract new URLs in it and follow them for crawling.
 
+|`transformer.from-headless-crawling.exclude-urls-regex`
+|Regular expression
+a |`.*(css\|gif\|gz\|ico\|jpeg\|jpg\|js\|mp3\|mp4\|pdf\|png\|svg\|zip\|woff2
+\|woff\|ttf\|doc\|docx\|xml\|json\|webmanifest)$`
+|The regular expression to match URLs.
+The crawler will not crawl all URLs that matching the given regular expression and they will not be added to the resulting table.
+https://en.wikipedia.org/wiki/URI_fragment[URI fragments] and https://en.wikipedia.org/wiki/Query_string[URL query] are ignored at filtering.
+
 |`transformer.from-headless-crawling.exclude-extensions-regex`
 |Regular expression
-a |`(css\|gif\|gz\|ico\|jpeg\|jpg\|js\|mp3\|mp4\|pdf\|png\|svg\|zip\|woff2
-\|woff\|ttf\|doc\|docx\|xml\|json\|webmanifest)`
-|The regular expression to match extensions in URLs.
+|no default value
+|Deprecated property, use `transformer.from-headless-crawling.exclude-urls-regex` instead. The regular expression to match extensions in URLs.
```

Comment:
please use admonitions for deprecations, see: https://docs.vividus.dev/vividus/latest/plugins/plugin-applitools.html#_general_properties